ext/phar/tar.c

```diff
@@ -1234,6 +1235,12 @@ int phar_tar_flush(phar_archive_data *phar, char *user_stub, zend_long len, int
 		entry.fp = php_stream_fopen_tmpfile();
 		if (entry.fp == NULL) {
 			spprintf(error, 0, "phar error: unable to create temporary file");
```

Comment:
Should this check if `error` is not null, too?

Reply:
No, we figured out these are pointless: https://github.com/php/php-src/pull/19356